fastlane/Fastfile

```diff
@@ -593,6 +593,37 @@ end
       SimCtl.create_device device, devicetype, runtime
     }
   end
+
+  #####################################################################################
+  # UI tests lanes
+  #####################################################################################
+
+    # Props to work done by others in this file and in
+    # https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/fastlane/Fastfile
+    # this is a combined rip-off from both
+
+    #####################################################################################
+    # run_UI_tests
+    # -----------------------------------------------------------------------------------
+    # Usage:
+    # bundle exec fastlane run_UI_tests [device:<Name of the iOS Simulator >]
+    #
+    # Example:
+    # bundle exec fastlane run_UI_tests device:"iPhone 12"
+    #####################################################################################
```

Comment:
Nitpick

```suggestion
  # Props to work done by others in this file and in
  # https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/fastlane/Fastfile
  # this is a combined rip-off from both

  #####################################################################################
  # run_UI_tests
  # -----------------------------------------------------------------------------------
  # Usage:
  # bundle exec fastlane run_UI_tests [device:<Name of the iOS Simulator >]
  #
  # Example:
  # bundle exec fastlane run_UI_tests device:"iPhone 12"
  #####################################################################################
```

I don't think we need a different indentation for this block. I've seen it in some `Fastfile` but I think it will soon be sorted out as we work on standardizing the format.

If we were to use an IDE to auto-indent this file, it would bring the indentation back one level.